core/src/SYCL/Kokkos_SYCL.cpp

```diff
@@ -81,6 +81,13 @@ SYCL::SYCL() : m_space_instance(&Impl::SYCLInternal::singleton()) {
       "SYCL instance constructor");
 }
 
+SYCL::SYCL(sycl::queue const stream)
+    : m_space_instance(new Impl::SYCLInternal) {
+  Impl::SYCLInternal::singleton().verify_is_initialized(
+      "SYCL instance constructor");
+  m_space_instance->initialize(stream);
```

Comment:
Any reason this isn't `m_space_instance->initialize(std::move(stream));` (other than `stream` is `const`)?

I know performance doesn't particularly matter in this case, I just find myself thinking "why copy?" in non-obvious cases.

Reply:
Honestly, I didn't put much consideration into it assuming that copying is cheap.